README.md

```diff
@@ -443,11 +443,11 @@ issuing stock is how companies often choose to do this.
   paper that prove you own the stock.
   - Sometimes you have stock but don’t have the physical certificate, as it may be held for
     you at a law office.
-    Some companies now manage their ownership through online services called **ownership
-    management platforms**, such as [Carta](https://carta.com/). If the company you work for uses
+    Some companies now manage their ownership through online services called *ownership
```

Comment:
This was also rephrased in #152 https://github.com/jlevy/og-equity-compensation/pull/152/files#diff-04c6e90faac2675aa89e2176d2eec7d8L443 so you'll have to resolve conflicts.